.circleci/unittest/windows/scripts/setup_env.sh

```diff
@@ -37,3 +37,9 @@ conda activate "${env_dir}"
 # 3. Install Conda dependencies
 printf "* Installing dependencies (except PyTorch)\n"
 conda env update --file "${this_dir}/environment.yml" --prune
+
+# 4. Downgrade setuptools on Python 3.7.
+#    See https://github.com/pytorch/vision/pull/5868
+if [[ "${PYTHON_VERSION}" == '3.7' ]]; then
+  pip install --upgrade setuptools==58.0.4
+fi
```

Comment:
Initially, `setuptools` and similar packages used [`cp3_10`](https://github.com/pypa/setuptools/blob/4fa006da4bbced755463b99914029c05c61523da/setuptools/_vendor/packaging/tags.py#L723-L729) as the interpreter tag for Python 3.10. After [PEP 641](https://peps.python.org/pep-0641/) was rejected, they changed to `cp310`. This patch was included in [`setuptools==59.1.0`](https://github.com/pypa/setuptools/blob/c5a4c42d38cd0ba5ecd34977e211ae4b6865afec/setuptools/_vendor/packaging/tags.py#L466-L467). With this tag set incorrectly, `python setup.py develop` ignores wheels on Python 3.10 and tries to use the source version. That is fine for pure python libraries, but means that it will try to compile Pillow from source.

Unfortunately, the `defaults` channel provides nothing between `setuptools==58.0.4` and `setuptools==61.2.0`. The former has the problems described above and the latter has problems on Python 3.7 (see #5868). Thus, I opted to use the latest release in general and special case Python 3.7 here. When we hard-depend on Python 3.8, we can simply remove this.